http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ChannelPipelineInitializer.java

```diff
@@ -125,7 +128,7 @@ private void configureHttp2(Channel ch, ChannelPipeline pipeline) {
         Http2FrameCodec codec =
             Http2FrameCodecBuilder.forClient()
                                   .headerSensitivityDetector((name, value) -> lowerCase(name.toString()).equals("authorization"))
-                                  .initialSettings(Http2Settings.defaultSettings().initialWindowSize(1_048_576))
+                                  .initialSettings(Http2Settings.defaultSettings().initialWindowSize(clientInitialWindowSize))
```

Comment:
Hi @dagnir  ,

I am doing a netty http2 based project based on aws sdk netty implementation.
The code works, now I am trying to tune window size and burst performance.

However, my frame size is limited to 64KB even if I change initial window size to 1MB(both server and client), set max frame size to 1MB and netty buffer size to 1MB. Do you have any suggestion for this case? 

Do I need to have a settingFrameHandler? (I thought Netty http2 automatically  process it.)

Here is a sample output of sending 1MB data from client to server.

```
[2020-04-20 11:31:07,363] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND SETTINGS: ack=false settings={INITIAL_WINDOW_SIZE=1000001, MAX_FRAME_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192} (server)
[2020-04-20 11:31:07,363] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND SETTINGS: ack=false settings={INITIAL_WINDOW_SIZE=1000002, MAX_FRAME_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192} (client)
[2020-04-20 11:31:07,364] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=1868932 (server)
[2020-04-20 11:31:07,365] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=1868934 (client)
[2020-04-20 11:31:07,380] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND SETTINGS: ack=false settings={INITIAL_WINDOW_SIZE=1000001, MAX_FRAME_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192} (client)
[2020-04-20 11:31:07,381] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND SETTINGS: ack=true (client)
[2020-04-20 11:31:07,382] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=1868932 (client)
[2020-04-20 11:31:07,382] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND SETTINGS: ack=false settings={INITIAL_WINDOW_SIZE=1000002, MAX_FRAME_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192} (server)
[2020-04-20 11:31:07,382] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND SETTINGS: ack=true (server)
[2020-04-20 11:31:07,383] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=1868934 (server)
[2020-04-20 11:31:07,383] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND SETTINGS: ack=true (server)
[2020-04-20 11:31:07,383] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND SETTINGS: ack=true (client)
[2020-04-20 11:31:07,397] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND HEADERS: streamId=3 headers=DefaultHttp2Headers[:method: POST, :scheme: https, :path: /] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false (client)
1049715
[2020-04-20 11:31:07,429] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=65519 bytes=0000000000100473000000040000000100000007636c69656e74310006100016340752000100000000000000001f8ddeae4cc4436ca7ff20dcf5a903c80003ff... (client)
[2020-04-20 11:31:07,431] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48797 bytes=84d5e53283c02b98ccdf0b05c954f956634fb356388e26f53c100ae67cd38783c5c2b4c71c96586c6384fa91eaf33b3981b5caf3f2fb588db510745c478a100c... (client)
[2020-04-20 11:31:07,432] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=49277 bytes=6d1c38c5d10857a7d979577b3f164fcf819d08d36354b59b6d8c56cb1a6599b7f55ec7d716f1f6ecd0007ed1eca481c10ca4313bb32e80bf1eab2a199198c0b4... (client)
[2020-04-20 11:31:07,433] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=7a7ec64a103729e71c51d9711fe959bc1155a5c4e13901f3cead3870f04032326b1a16c3f690f222dd643bbcd970013da5ea49fcb37d669b2e28e169a3692289... (client)
[2020-04-20 11:31:07,433] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=cd3fac52b7fcf37be5476c88a65d399ff056b2397a8dab6b692866c8dff882731352139bf2436d665ff377f187548a0fab6ecf8042084660a8c3f6b1449b838d... (client)
[2020-04-20 11:31:07,434] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=c944383ca89c6de1dc660e347bcef000811b7cb1fca036c13d73ffb9588f86eaaf21e9ccfc4850f0000fc5448843d436512a033d36a15ae7538c3198725a76ab... (client)
[2020-04-20 11:31:07,434] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND HEADERS: streamId=3 headers=DefaultHttp2Headers[:method: POST, :scheme: https, :path: /] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false (server)
[2020-04-20 11:31:07,435] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=18ce2bc93318631d8bc931dda301c17f9ccead375d046e46f1dbeaadbdc35b6d434f9169ce4d1114210ec92d4fc54fe4ce02c863b4e1cec5c41f4b8eb85eb8ff... (client)
[2020-04-20 11:31:07,435] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=4c69a54cbc92db05bc65cecedf9d78bdc6b775a0d9a679cf9d33809c792bb425a1a034179dc31afe6a9f15c0addfad497dbc89cce993068e5bf1b12bbd22f1c1... (client)
[2020-04-20 11:31:07,436] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=296f12be797c2e3d3c3618b06d98c5c6fdd6a75cf7aa4187df61f437467411c5eab5db0ba483c9f7b7dc5b2b050061f89e5f19ae8c5dd18af03878ff2dffea5f... (client)
[2020-04-20 11:31:07,437] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=3ea9f3aabd6fa02723b02a720c0ab7e132df7668cb21ad7cde3f486b2bbfafbdf56b16699206f17955f563eb5c80c56e4e3705547be2549c691a59a991922837... (client)
[2020-04-20 11:31:07,437] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=f36fad036afd4661d50a5a445a6c2a422a52348de8164ad2ad4fce3224223ee7a069d09a58206cc6f264aa2497f9408bd2e5f2706dfb5c3ab2359fcf6f95a082... (client)
[2020-04-20 11:31:07,437] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=6e8c1f7844147e66619be97b84d67bd104d822b2b178f06d4ce1f2f41d318797fa8f8eb6e72ab432b9cbcbde05222da670b3da3cc93847f3fefa23abc1fcee18... (client)
[2020-04-20 11:31:07,438] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=3f8df5afc121b5a8b0985eff2a3c52a6570f90a95512ee87dd30fe31c62823287e96e26b7d28e3e653f2c12fc456d8cd52d13be84b96402882cc2099e4be075b... (client)
[2020-04-20 11:31:07,438] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=9543b6bc95ff5ba1c4646bcea8a785902188f7edd4380af147adef3e45a9b22336a77f93405c0a31eea1b3715e6efa752e06560933ddb8fc456a43831962422a... (client)
[2020-04-20 11:31:07,439] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=730ec6ca97906f528620a974420a56ceebfe866c7703b3dc4ef61b1424212ecc2489d9ba0bfc6c26dfaa0d6721e232b0ad105e4be8952bb0a423921398d63c3f... (client)
[2020-04-20 11:31:07,439] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=5a56e449e0616a467092a37e4f814918ce6de543c220ef702006eb8f814f2e22182d03af334ee7213d4c655928cb34f380dc7a383fa1c0654655a931e3f746f2... (client)
[2020-04-20 11:31:07,440] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=f0514b3284ae8778ac275890fa84d0e843064d3cd09a81232a60ce6689103f3064ba869670818be51e23490a283cc94d3e2af2727721dc617d3cdfa6c5c00dc1... (client)
[2020-04-20 11:31:07,441] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=dbfbb722e0c74c869c8574961c65b302735c3e0a25d587402be45eb82f9e0965ab411336ade15dc6f4585d6b2065b090ab13c2e620bef71803873c49dcf2bd6e... (client)
[2020-04-20 11:31:07,441] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=0b308f281a6502275c4c4b448fdbf24ceabde1addc7b15c89eb7c5f9f9ecd524d7f939f2465b7f987acc1ad64e1ea46b2cf3712a98f248b2ea39b0baf2c917e6... (client)
[2020-04-20 11:31:07,442] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=1490c576c06368f063f8200ed2d0bc1e80a09859e23ecf6d1f97935d91f644c2c555e28087260126b745b2606d2ec40e9d99f8cb64bac329aeb3c179e667fd11... (client)
[2020-04-20 11:31:07,442] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=e396cd257ec1ca99499109ba6287f4b82330f3e3487a2da4d77cd7aad7fd48bb50fb56b7fade7d62e27ff6b538f2d65c8733d33b07e0c20182f7d5d10e2a77f0... (client)
[2020-04-20 11:31:07,443] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=1cb2aadb99483c47b2b961cf0a4e9cc85fceba19f7fea4f55ad37aee5a3be399a7edaca72999bdeb54f93ae606793f5de3492da178cf803a20050e0072e3129c... (client)
[2020-04-20 11:31:07,443] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=f45bc3968b627271571889d3a86ccbc8a67ebdd4261f72cdc77014c29963be8bc352a3b6a7426fc1ecb910753a318f4eb273e0672d9693f906622d55c8ecd087... (client)
[2020-04-20 11:31:07,443] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=false length=20758 bytes=6644769ea9b21b2a9c57531e26095b2ae56e5487dffa67ad74f4a3a5921ded162e99b84d3da5feb8bd9f80f5a2d6f6645513be2ea0da95015c648955d917ed73... (client)
[2020-04-20 11:31:07,444] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=65519 bytes=0000000000100473000000040000000100000007636c69656e74310006100016340752000100000000000000001f8ddeae4cc4436ca7ff20dcf5a903c80003ff... (server)
[2020-04-20 11:31:07,447] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48797 bytes=84d5e53283c02b98ccdf0b05c954f956634fb356388e26f53c100ae67cd38783c5c2b4c71c96586c6384fa91eaf33b3981b5caf3f2fb588db510745c478a100c... (server)
[2020-04-20 11:31:07,448] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=49277 bytes=6d1c38c5d10857a7d979577b3f164fcf819d08d36354b59b6d8c56cb1a6599b7f55ec7d716f1f6ecd0007ed1eca481c10ca4313bb32e80bf1eab2a199198c0b4... (server)
[2020-04-20 11:31:07,448] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=7a7ec64a103729e71c51d9711fe959bc1155a5c4e13901f3cead3870f04032326b1a16c3f690f222dd643bbcd970013da5ea49fcb37d669b2e28e169a3692289... (server)
[2020-04-20 11:31:07,449] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=cd3fac52b7fcf37be5476c88a65d399ff056b2397a8dab6b692866c8dff882731352139bf2436d665ff377f187548a0fab6ecf8042084660a8c3f6b1449b838d... (server)
[2020-04-20 11:31:07,449] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=c944383ca89c6de1dc660e347bcef000811b7cb1fca036c13d73ffb9588f86eaaf21e9ccfc4850f0000fc5448843d436512a033d36a15ae7538c3198725a76ab... (server)
[2020-04-20 11:31:07,450] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=18ce2bc93318631d8bc931dda301c17f9ccead375d046e46f1dbeaadbdc35b6d434f9169ce4d1114210ec92d4fc54fe4ce02c863b4e1cec5c41f4b8eb85eb8ff... (server)
[2020-04-20 11:31:07,450] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=4c69a54cbc92db05bc65cecedf9d78bdc6b775a0d9a679cf9d33809c792bb425a1a034179dc31afe6a9f15c0addfad497dbc89cce993068e5bf1b12bbd22f1c1... (server)
[2020-04-20 11:31:07,450] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=296f12be797c2e3d3c3618b06d98c5c6fdd6a75cf7aa4187df61f437467411c5eab5db0ba483c9f7b7dc5b2b050061f89e5f19ae8c5dd18af03878ff2dffea5f... (server)
[2020-04-20 11:31:07,451] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=3ea9f3aabd6fa02723b02a720c0ab7e132df7668cb21ad7cde3f486b2bbfafbdf56b16699206f17955f563eb5c80c56e4e3705547be2549c691a59a991922837... (server)
[2020-04-20 11:31:07,451] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=f36fad036afd4661d50a5a445a6c2a422a52348de8164ad2ad4fce3224223ee7a069d09a58206cc6f264aa2497f9408bd2e5f2706dfb5c3ab2359fcf6f95a082... (server)
[2020-04-20 11:31:07,451] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=6e8c1f7844147e66619be97b84d67bd104d822b2b178f06d4ce1f2f41d318797fa8f8eb6e72ab432b9cbcbde05222da670b3da3cc93847f3fefa23abc1fcee18... (server)
[2020-04-20 11:31:07,452] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=3f8df5afc121b5a8b0985eff2a3c52a6570f90a95512ee87dd30fe31c62823287e96e26b7d28e3e653f2c12fc456d8cd52d13be84b96402882cc2099e4be075b... (server)
[2020-04-20 11:31:07,453] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=9543b6bc95ff5ba1c4646bcea8a785902188f7edd4380af147adef3e45a9b22336a77f93405c0a31eea1b3715e6efa752e06560933ddb8fc456a43831962422a... (server)
[2020-04-20 11:31:07,453] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=730ec6ca97906f528620a974420a56ceebfe866c7703b3dc4ef61b1424212ecc2489d9ba0bfc6c26dfaa0d6721e232b0ad105e4be8952bb0a423921398d63c3f... (server)
[2020-04-20 11:31:07,453] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND WINDOW_UPDATE: streamId=3 windowSizeIncrement=652983 (server)
[2020-04-20 11:31:07,454] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=5a56e449e0616a467092a37e4f814918ce6de543c220ef702006eb8f814f2e22182d03af334ee7213d4c655928cb34f380dc7a383fa1c0654655a931e3f746f2... (server)
[2020-04-20 11:31:07,454] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=f0514b3284ae8778ac275890fa84d0e843064d3cd09a81232a60ce6689103f3064ba869670818be51e23490a283cc94d3e2af2727721dc617d3cdfa6c5c00dc1... (server)
[2020-04-20 11:31:07,454] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=dbfbb722e0c74c869c8574961c65b302735c3e0a25d587402be45eb82f9e0965ab411336ade15dc6f4585d6b2065b090ab13c2e620bef71803873c49dcf2bd6e... (server)
[2020-04-20 11:31:07,455] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=0b308f281a6502275c4c4b448fdbf24ceabde1addc7b15c89eb7c5f9f9ecd524d7f939f2465b7f987acc1ad64e1ea46b2cf3712a98f248b2ea39b0baf2c917e6... (server)
[2020-04-20 11:31:07,455] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=1490c576c06368f063f8200ed2d0bc1e80a09859e23ecf6d1f97935d91f644c2c555e28087260126b745b2606d2ec40e9d99f8cb64bac329aeb3c179e667fd11... (server)
[2020-04-20 11:31:07,456] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=e396cd257ec1ca99499109ba6287f4b82330f3e3487a2da4d77cd7aad7fd48bb50fb56b7fade7d62e27ff6b538f2d65c8733d33b07e0c20182f7d5d10e2a77f0... (server)
[2020-04-20 11:31:07,456] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=48672 bytes=1cb2aadb99483c47b2b961cf0a4e9cc85fceba19f7fea4f55ad37aee5a3be399a7edaca72999bdeb54f93ae606793f5de3492da178cf803a20050e0072e3129c... (server)
[2020-04-20 11:31:07,456] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=32893 bytes=f45bc3968b627271571889d3a86ccbc8a67ebdd4261f72cdc77014c29963be8bc352a3b6a7426fc1ecb910753a318f4eb273e0672d9693f906622d55c8ecd087... (server)
[2020-04-20 11:31:07,456] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=979243 (server)
[2020-04-20 11:31:07,457] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=false length=20758 bytes=6644769ea9b21b2a9c57531e26095b2ae56e5487dffa67ad74f4a3a5921ded162e99b84d3da5feb8bd9f80f5a2d6f6645513be2ea0da95015c648955d917ed73... (server)
[2020-04-20 11:31:07,457] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND WINDOW_UPDATE: streamId=3 windowSizeIncrement=652983 (client)
[2020-04-20 11:31:07,457] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=979243 (client)
[2020-04-20 11:31:07,457] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND DATA: streamId=3 padding=0 endStream=true length=49714 bytes=2af5f643a8414096775219030073d04afad4f2b2a89d5693dac1c8411998b61914e7c4e6a9bd3f719a48c20a82fae080476dc1fb90083740272eb7dcbd94a7e9... (client)
[2020-04-20 11:31:07,458] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND DATA: streamId=3 padding=0 endStream=true length=49714 bytes=2af5f643a8414096775219030073d04afad4f2b2a89d5693dac1c8411998b61914e7c4e6a9bd3f719a48c20a82fae080476dc1fb90083740272eb7dcbd94a7e9... (server)
[2020-04-20 11:31:07,515] INFO Creating first segment with start offset Name = [] Offset = [0] (com.github.ambry.store.PersistentIndex)
[2020-04-20 11:31:07,524] INFO Chunk size: 29 (com.github.ambry.rest.NettyResponseChannel)
[2020-04-20 11:31:07,525] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND HEADERS: streamId=3 headers=DefaultHttp2Headers[:status: 200, content-length: 29] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false (server)
[2020-04-20 11:31:07,527] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND HEADERS: streamId=3 headers=DefaultHttp2Headers[:status: 200, content-length: 29] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false (client)
[2020-04-20 11:31:07,528] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] OUTBOUND DATA: streamId=3 padding=0 endStream=true length=29 bytes=000000000000001d000100010000000100000007636c69656e74310000 (server)
[2020-04-20 11:31:07,529] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] INBOUND DATA: streamId=3 padding=0 endStream=true length=29 bytes=000000000000001d000100010000000100000007636c69656e74310000 (client)
[2020-04-20 11:31:07,529] INFO fame size: 29 (com.github.ambry.network.http2.Http2ClientStreamStatsHandler)
[2020-04-20 11:31:07,529] INFO frame(data) count is 2 (com.github.ambry.network.http2.Http2ClientStreamStatsHandler)
[2020-04-20 11:31:07,529] INFO [id: 0xcb379da5, L:/127.0.0.1:60952 - R:localhost/127.0.0.1:64001] OUTBOUND RST_STREAM: streamId=3 errorCode=8 (client)
[2020-04-20 11:31:07,530] INFO [id: 0x02eee363, L:/127.0.0.1:64001 - R:localhost/127.0.0.1:60952] INBOUND RST_STREAM: streamId=3 errorCode=8 (server)
```